src/main/java/nl/novi/techiteasy1121/services/TelevisionService.java

```diff
@@ -167,6 +162,44 @@ public TelevisionDto transferToDto(Television television){
         dto.setOriginalStock(television.getOriginalStock());
         dto.setSold(television.getSold());
 
+        // Als extra op deze transfer methode, voegen we ook de relaties toe.
+        // Hier moeten we eerst een null check voor doen,
+        // omdat we anders in CIModule.transferToDto de get-methodes van "null" aanroepen en dat kan niet.
+        if(television.getCiModule() != null){
+            dto.setCiModuleDto(CIModuleService.transferToDto(television.getCiModule()));
+        }
+
+
         return dto;
     }
+
+    public void assignRemoteControllerToTelevision(Long id, Long remoteControllerId) {
+        var optionalTelevision = televisionRepository.findById(id);
+        var optionalRemoteController = remoteControllerRepository.findById(remoteControllerId);
```

Comment:
De methoden in de TelevisionService zijn goed georganiseerd, maar let op dat je niet te veel nested calls maakt. Probeer de logica te scheiden waar mogelijk.